### DISCUSSION TITLE
Up and running and at 100% - why doesn't my battery charge fall?

### DISCUSSION THREAD

**Discussion by fearby** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1247] [Q&A]
Al dente pro Sonoma MacBook Pro M3 - Used the helper application to set it to go to 70% but using external monitor via USBC I am at 100% charge and show no sign of dropping. Is this what al dente is supposed to do?

**Comment by marcospgp** [comment]
Your mac is likely relying on the charger and thus the battery isn't being relied on. Try unplugging the charger until it goes below your limit.

**Reply by fearby** [reply]
The charger is not plugged in. I am using an external monitor via USBC and that is where the charge is coming from. Does AL dente not work if USBC devices are plugged in and providing charge?

**Comment by MatthiasKerbl** [comment]
Hi @fearby,

If you want your MacBook battery to stay at 70% instead of 100%, you must first discharge it to 70%. You can do this by using the Discharge feature of AlDente or unplugging your MacBook from all power supplies.

Best,
Matthias
